lib/coinbase/address/wallet_address.rb

```diff
@@ -96,39 +96,54 @@ def trade(amount, from_asset_id, to_asset_id)
     end
 
     # Stakes the given amount of the given Asset
+    # Timeout::Error if the Staking Operation takes longer than the given timeout.
```

Comment:
Generally this should be documented like such:

```
# @raise [Timeout::Error]
```

Reply:
Yup fixed - most like copied from trade/transfer